README.md

```diff
@@ -4,17 +4,18 @@
 
 #### Frameworks performance comparison.
 
-**ratrace** is part of the **hapi** ecosystem and was designed to work seamlessly with the [hapi web framework](https://hapi.dev) and its other components (but works great on its own or with other frameworks). If you are using a different web framework and find this module useful, check out [hapi](https://hapi.dev) � they work even better together.
+**ratrace** is part of the **hapi** ecosystem and was designed to work seamlessly with the [hapi web framework](https://hapi.dev) and its other components (but works great on its own or with other frameworks). If you are using a different web framework and find this module useful, check out [hapi](https://hapi.dev) – they work even better together.
```

Comment:
What changed on this line and line 21? If it wasn't intentional, can you undo it?